locales/en.json

```
@@ -102,7 +102,7 @@
   "mail.webhook.payment.unapproved.subject": "The payment for a task that you made on Gitpay wasn't approved",
   "mail.webhook.payment.unapproved.message": "<p>The payment of $ {{amount}} for a task on Gitpay wasn't approved</p><p>Reason: {{reason}}</p>.",
   "mail.webhook.payment.transfer.subject": "There's a Gitpay transfer on the way!",
-  "mail.webhook.payment.transfer.message": "<p>A transfer of $ {{amount}} was sent to your account and we will let you know when it's concluded.</p><p>This transfer is about the task <a href='{{url}}'>{{url}}</a> that you concluded.</p>",
+  "mail.webhook.payment.transfer.message": "<p>A transfer of $ {{amount}} was sent to your account and we will let you know when it's concluded.</p><p>This transfer is about the task <strong>{{title}}</strong> <a href='{{url}}'>{{url}}</a> that you concluded.</p>",
```

Maybe it will be better like this:

```suggestion
  "mail.webhook.payment.transfer.message": "<p>A transfer of $ {{amount}} was sent to your account and we will let you know when it's concluded.</p><p>This transfer is about the task <a href='{{url}}'>{{title}}</a> that you concluded.</p>",
```